elasticsearch/initialize_es_index.py

```diff
@@ -0,0 +1,53 @@
+"""
+This script generates the set of empty indices
+1. es_index_prefix (CLI parameters)
+2. type of the data (defined in the global variable TYPES
+"""
+
+from elasticsearch import Elasticsearch
+import os
+import click
+
+
+TYPES = ["organism", "specimen", "dataset", "experiment", "file"]
+
+
+# use click library to get command line parameters
+@click.command()
+@click.option(
+    '--es_host',
+    default="http://wp-np3-e2:9200",
+    help='Specify the Elastic Search server (port should be included), default to be http://wp-np3-e2:9200.'
```

Comment:
Do we really need to have ES server address in parameters? It's quite unlikely that we going to change it in observable feature.

Reply:
not really, but better to have this option just in case. And there is a default value provided for es_host. If you do not want to type, it would be fine. This option also makes it to have same parameter as the old perl version

Reply:
Ok, let's have it.